src/components/Chart.jsx

```diff
@@ -63,9 +63,18 @@ class Chart extends Component {
         const insideHolder = array.filter(c => !/(TradeStart)|(TradeEnd)/.test(c.type.displayName));
         const insideSubHolder = array.filter(c => /(TradeStart)|(TradeEnd)/.test(c.type.displayName));
         const renderTopWidgets = topWidgets || defaultTopWidgets;
+        const contextClassName = () =>{
```

Comment:
this function can be simplified. In fact you may not even need it with template literals